### PR TITLE
added AncientBeast.com to the list of projects

### PR DESCRIPTION
**Pull request by DreadKnight**
I'm adding my open source project, Ancient Beast, https://AncientBeast.com to the list of project using the repository
https://github.com/FreezingMoon/AncientBeast

This PR fixes https://github.com/firstcontributions/first-contributions/issues/60079

### CODE REVIEW

File: src/components/ProjectList/listOfProjects.js [hunk 1]
```diff
@@ -1,12 +1,18 @@
 const projectList = [
+   {
+    name: 'Ancient Beast',
+    imageSrc: 'https://raw.githubusercontent.com/FreezingMoon/AncientBeast-Website/master/favicon.png',
+    projectLink: 'https://github.com/FreezingMoon/AncientBeast',
```
Comment (Roshanjossey) [inline src/components/ProjectList/listOfProjects.js:5]: For other projects, we take users to `/contribute` path so they can see issues labeled `good-first-issue` or `help wanted`. For AncientBeast, I think the similar thing would be to point to issues with label `easy`

**Review by Roshanjossey — CHANGES_REQUESTED** [review]
I've added a suggestion. I've got one question too. Did you put it in the top of the list so it'd show up on the top of the website?

File: src/components/ProjectList/listOfProjects.js (hunk 1 above)
Comment (DreadKnight) [reply src/components/ProjectList/listOfProjects.js:5]: I've noticed that for some of the other projects. Was thinking if to do that as well, though directly linking to the repo makes the readme more exposed and the contribution guide. Either way is fine with me though 👍🏻

**Comment by DreadKnight** [comment]
> I've added a suggestion. I've got one question too. Did you put it in the top of the list so it'd show up on the top of the website?

Resolved that suggestion. No, I thought that would have been nice but can't impose; usually expecting lists to either be alphabetical or not really matter. Was thinking the website has some different sorting algorithm and doesn't respect the order of the file, but I might be wrong 🐻

**Comment by Roshanjossey** [comment]
>  Was thinking the website has some different sorting algorithm and doesn't respect the order of the file, but I might be wrong 🐻

It's random

**Comment by DreadKnight** [comment]
> > Was thinking the website has some different sorting algorithm and doesn't respect the order of the file, but I might be wrong bear
> 
> It's random

Good 🎲

**Comment by Roshanjossey** [comment]
I went through some issue, PR threads at AncientBeast repo. Keep up the good work. Thank you for adding the project here

**Comment by DreadKnight** [comment]
> I went through some issue, PR threads at AncientBeast repo. Keep up the good work. Thank you for adding the project here

Woot! Thanks for taking the time to take a peek at it and merging this, will hopefully get more eyes on the project, every bit helps!
I've closed https://github.com/FreezingMoon/AncientBeast/issues/1903 🥳

**Pull request merged**